src/dashboard/src/components/backlog/views.py

```diff
@@ -185,14 +195,27 @@ def search(request):
 
     results = [x["_source"] for x in hits["hits"]["hits"]]
 
+    def format_values(k, v):
+        """
+        Format search result values for display in Backlog tab.
+        """
+        if k == "size":
+            return "{0:.2f} MB".format(float(v))
+        return v
```

Comment:
Good to know! I'll bookmark that. I was trying to make the `size` property in the `transfers` index match how it's handle in the `aips` index, where it is [stored as a float that represents the size in MBs](https://github.com/artefactual/archivematica/blob/4f2abe13a40ae330462a01d38e1ab5b3ce46d950/src/archivematicaCommon/lib/elasticSearchFunctions.py#L450). So the formatting is about stripping off the trailing decimals rather than converting from bytes to another unit. My thinking was that since we don't want to force users with lots of data to re-index it's unlikely that we'll change how we're storing this in the `aips` index, and so we may as well make the two indices consistent. However I'm happy to make changes if you think it'd be better to handle differently.

Reply:
I discussed this briefly with the project analyst today and the decision was to take the question up with the product manager. I'll do that and update here when there's an answer.

Note to self: It should be possible to use [HTML5 data attributes](https://www.datatables.net/release-datatables/examples/advanced_init/html5-data-attributes.html) to keep sorting functional even if we auto-format the display using `filesizeformat`.

If we do create this sort of "display dependency" I will implement as a shared helper. But perhaps that's not necessary.

Reply:
After internal conversation, I've made changes to store size in the `transfers` index as bytes ("long" type in ES), and format the data for the DataTable using `filesizeformat`.